contracts/FlowFees.cdc

```diff
@@ -9,6 +9,12 @@ pub contract FlowFees {
     // Event that is emitted when tokens are withdrawn from the fee vault
     pub event TokensWithdrawn(amount: UFix64)
 
+    // Event that is emitted when fees are deducted
+    pub event FeesDeducted(amount: UFix64, inclusionEffort: UFix64, executionEffort: UFix64)
```

Comment:
Would it make sense to just add these parameters to the end of the TokensDeposited Event so there isn't the breaking change?

Reply:
I'm also tempted to add an `Address` parameter here

Reply:
> Would it make sense to just add these parameters to the end of the TokensDeposited Event so there isn't the breaking change?
I was thinking about that too, but that event is also used for account creation, which does not have the extra parameters. We could emit both events, but that increases the event count for all transactions. Another option would perhaps be to create a separate event for account creation instead.

> I'm also tempted to add an Address parameter here
That address will be the address of the payer which you have access to if you have the transaction data, so I'm not sure that adding the address here makes things better for users.

Reply:
> That address will be the address of the payer which you have access to if you have the transaction data, so I'm not sure that adding the address here makes things better for users.
Good point. I didn't think about that

> We could emit both events, but that increases the event count for all transactions. Another option would perhaps be to create a separate event for account creation instead.
It really depends on how many people are monitoring the event and if it would affect them. my guess is that it isn't very many, so adding this new one is probably fine

Reply:
So you are ok with only triggering `FeesDeducted`?

Reply:
yeah, I'm ok with that